tests/integration/update_cluster/karpenter/data/aws_s3_object_minimal.example.com-addons-karpenter.sh-k8s-1.19_content

```diff
@@ -2050,15 +4332,47 @@ metadata:
     k8s-addon: karpenter.sh
   name: karpenter-nodes-default
 spec:
-  launchTemplate: karpenter-nodes-default.minimal.example.com
-  subnetSelector:
-    kops.k8s.io/instance-group/karpenter-nodes-default: '*'
-    kubernetes.io/cluster/minimal.example.com: '*'
+  amiFamily: Custom
+  amiSelectorTerms:
+  - id: ami-12345678
+  blockDeviceMappings:
+  - deviceName: /dev/xvda
+    ebs: {}
```

Comment:
looks like the awsmodel's default root volume fields aren't being set on the InstanceGroup that we're referencing here so we'll need to reuse that defaulting logic.